README.rst

```diff
@@ -73,6 +73,16 @@ Some Python HTTP clients that you can use to retrieve HTML include `requests`_,
 .. _urllib.request module: https://docs.python.org/3/library/urllib.request.html
 
 
+Test data
+---------
+
+As a precondition for inclusion in this library, each recipe scraper is required to include test coverage to demonstrate that it can accurately retrieve recipe information from the website it relates to.
+
+To achieve that, the development source repository includes a minimal number of test HTML files from the public web -- often only one or two per recipe website -- to confirm the scraper's behaviour.  These test files are not included in the [published releases](https://pypi.org/project/recipe-scrapers/) of the library.
+
+If you are the author/publisher of any of those files and would prefer for them to be removed, please contact one of the library's authors and/or file a bugreport -- preferably from an email address associated with the recipe website, or otherwise proving ownership of the content.  Note that this reduces our ability to support affected website(s).
```

Comment:
```suggestion
If you are the author/publisher of any of those files and would prefer for them to be removed, please contact the library's authors and/or file a bugreport -- preferably from an email address associated with the recipe website, or otherwise proving ownership of the content you'd like us to remove.  Note that removal of test HTML will reduce our ability to support the affected website(s).
```